serverless/src/functions/builds/handler.ts

```diff
@@ -27,7 +27,8 @@ export const submitBuildInfo = async (
       ipfsHash: data.ipfsHash,
       domain: data.domain,
     };
-
+    console.log(buildInfo);
```

Comment:
maybe leftover

Reply:
I'm not quite sure if we want to keep this or not. But I guess we can add it whenever needed, later.